src/GraknClient.jl

```diff
@@ -6,9 +6,10 @@ In root folder run $ bash setup.sh to genrate the proto files
 then install the packages from GitHub
 Pkg.rm("gRPCClient")
 Pkg.add(url="https://github.com/JuliaComputing/gRPCClient.jl")
```

Comment:
Why do both add and dev, doesn't `]dev https://github.com/JuliaComputing/gRPCClient.jl` work as well?